packages/plugin-stripe/src/index.ts

```diff
@@ -59,7 +59,7 @@ export const stripePlugin =
       })
     }
 
-    for (const collection of collections) {
+    for (const collection of collections!) {
```

Comment:
I'd rather avoid the non-null assertion and do something like `(collections ?? [])`

Reply:
Agreed on not changing runtime behavior to reach strict. As long as this is not the norm, I'm fine with it.